R/pkgdown.R

```diff
@@ -1,14 +1,19 @@
 #' Use pkgdown
 #'
 #' [pkgdown](https://github.com/hadley/pkgdown) makes it easy to turn your
-#' package into a beautiful website. This helper creates `_pkgdown.yml`
-#' and `docs/` for you, and adds them to `.Rbuildignore`
+#' package into a beautiful website. This helper creates a pkgdown config file
+#' and a directory for pkgdown docs, and adds them to `.Rbuildignore`
+#'
+#' @seealso http://pkgdown.r-lib.org/articles/pkgdown.html#configuration
+#'
+#' @param config_file pkgdown yaml config file
+#' @param dst_path target directory for pkgdown docs
 #'
 #' @export
-use_pkgdown <- function() {
+use_pkgdown <- function(config_file = "_pkgdown.yml", dst_path = "docs") {
```

Comment:
If the user sets `dst_path` should it be reflected in the generated `pkgdown.yml` file?

Reply:
Yeah pkgdown takes:

```yaml
destination: docs
```

So we'd need to write that line into the created `_pkgdown.yml` file.

Reply:
What's the recommended way to modify this file? Something like:

```r
cat(paste("destination:", dst_dir), file = config_file)
```

Reply:
Although it means something different, other functions in usethis this have a `destdir` argument FWIW (`create_*()`, `use_course()`).

Reply:
@jayhesselberth  I think we can assume that this function creates the file, so you can use `write_over()`

And make the argument either `destination` (to match pkgdown) or `destdir` (to match usethis). (I don't have strong feelings which)